docker/compose.yml

```diff
@@ -4,15 +4,24 @@ services:
     container_name: trezor-user-env.unix
     image: registry.gitlab.com/satoshilabs/trezor/trezor-user-env/trezor-user-env
     network_mode: "host"
+    build:
+      context: ..
+      dockerfile: ./docker/Dockerfile
```

Comment:
This is just a leftover right? Because I would still build via `pull` from the registry.